lib/fizzbuzz.rb

```diff
@@ -1,9 +1,20 @@
 class SuperFizzBuzz
 
   def run(input)
-
     #Implement your code here
-
+    if(input.is_a? Integer)
```

Comment:
if you really want to type-check the input, then use a guard clause: something like
```
return "Input isn't an integer" unless input.is_a? Integer
```
at the top.  The helps to minimize the complexity you're asking human readers to keep in mind to understand your code.